runner/cf-driver/prepare.sh

```diff
@@ -142,16 +174,26 @@ start_services () {
        return
     fi
 
-    for l in $(echo "$ci_job_services" | jq -rc '.[]'); do
+    services=$(jq -rc '.services[]' "$JOB_RESPONSE_FILE")
```

Comment:
[question] where does `$JOB_RESPONSE_FILE` come from?

Reply:
It's made available by the gitlab-runner process: https://docs.gitlab.com/runner/executors/custom.html#job-response. I should add a note with the doc link.

My understanding is it's the full JSON response that gitlab-runner is getting when it communicates with gitlab.com to get the job info.